package.json

```diff
@@ -42,7 +42,6 @@
   ],
   "dependencies": {
     "chalk": "^1.0.0",
-    "chg": "^0.3.2",
```

Comment:
I actually need this in `master` now for an official CHANGELOG. Can this change be undone?